apps/etl/etl_tasks/pdc.py

```diff
@@ -45,13 +45,15 @@ def extract_and_transform_pdc_data():
     )
 
 
-def extract_and_transform_historical_pdc_data():
-    pdc_start_date = datetime.strptime(str(etl_config.PDC_START_DATE), "%Y-%m-%d")
+def extract_and_transform_historical_pdc_data(
+    start_date: date,
+    end_date: date,
+):
+    pdc_start_date = datetime.strptime(str(start_date), "%Y-%m-%d")
     pdc_interval_years = etl_config.PDC_EXTRACTION_INTERVAL_YEARS
 
     pdc_end_date = pdc_start_date.replace(year=pdc_start_date.year + pdc_interval_years)
-    if pdc_end_date > datetime.now():
-        pdc_end_date = datetime.now()
+    end_date = datetime.strptime(str(end_date), "%Y-%m-%d")
```

Comment:
@sudan45  end_date is not used; in historical extraction, we need to set the limit as end_date and not the current_date which is done below. Please kindly fix it.